apps/kilocode-docs/docs/features/auto-approving-actions.md

```diff
@@ -30,6 +30,7 @@ _Prompt text box and Expanded toolbar with all options_
 | ------------------------------ | ------------------------------------------------ | ----------- |
 | **Read files and directories** | Lets Kilo Code access files without asking       | Medium      |
 | **Edit files**                 | Lets Kilo Code modify files without asking       | **High**    |
+| **Delete files**               | Lets Kilo Code delete files without asking       | **High**    |
```

Comment:
We don't have a separate approval settings toggle for this. This currently falls under the edit files permission.